.ci/ci_clh_entry_point.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script is called by our jenkins instances, triggered by PRs on Cloud Hypervisor.
+# It relies on the following environment variables being set:
+# REPO_OWNER    - owner of the source repository (default: cloud-hypervisor)
+# REPO_NAME     - repository name (default: cloud-hypervisor)
+# PULL_BASE_REF - name of the branch where the pull request is merged to (default: main)
+# PULL_NUMBER   - pull request number (REQUIRED)
+#
+# (see: http://jenkins.katacontainers.io/job/kata-containers-2-clh-PR/)
+#
+# Usage:
+# curl -OL https://raw.githubusercontent.com/kata-containers/tests/main/.ci/ci_clh_entry_point.sh
+# bash ci_clh_entry_point.sh
+
+set -o errexit
+set -o pipefail
+set -o errtrace
```

Comment:
Could you add the following for extra safety and debugability:

```bash
 set -o nounset
[ -n "${DEBUG:-}" ] && set -o xtrace
```

Reply:
Done!